test/cmdlineTests/yul_optimizer_steps_without_optimize_empty_sequence/input.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
+
+contract C
+{
+    constructor() {}
+
+    function foo() public pure returns (bool)
+    {
+        uint a = 100;
+        uint b = a;
+        uint c = a;
```

Comment:
I expanded this test by introducing unused variables to confirm that the unused pruner is not run (which can be seen in the output file).

Reply:
Would be better to put that info as a comment in the test rather than in the review :)